src/OpenTK.Windowing.Desktop/GameWindow.cs

```diff
@@ -105,7 +108,7 @@ public class GameWindow : NativeWindow
         ///     Do not enable this unless your code is thread safe.
         ///   </para>
         /// </remarks>
-        [Obsolete("There is not one size fits all multithreading solution, especially for OpenGL. This option will be removed in future versions, and you will have to implement what you need instead.")]
+        [Obsolete("There is not one size fits all multithreading solution, especially for OpenGL. This feature has been removed and will not work.", true)]
         public bool IsMultiThreaded { get; }
```

Comment:
Let's have this throw an exception on `get` as it has no meaning now.

Reply:
Trying to read this property directly is a compile error, not sure we need to make getting the property an exception, but if you really think we should I can add that.

Reply:
All good then as-is.